# Generated by Django 3.2.3 on 2025-04-18 23:59

import django.contrib.auth.validators
import django.core.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('users', '0001_initial'),
    ]

    operations = [
        migrations.AlterModelManagers(
            name='customuser',
            managers=[
            ],
        ),
        migrations.AlterField(
            model_name='customuser',
            name='first_name',
            field=models.CharField(max_length=150, validators=[django.core.validators.RegexValidator(code='invalid_name', message='Поле должно содержать только буквы', regex='^[А-Яа-яЁёA-Za-z]+$')], verbose_name='Имя'),
        ),
        migrations.AlterField(
            model_name='customuser',
            name='last_name',
            field=models.CharField(max_length=150, validators=[django.core.validators.RegexValidator(code='invalid_name', message='Поле должно содержать только буквы', regex='^[А-Яа-яЁёA-Za-z]+$')], verbose_name='Фамилия'),
        ),
        migrations.AlterField(
            model_name='customuser',
            name='username',
            field=models.CharField(error_messages={'unique': 'Никнейм занят.'}, max_length=150, unique=True, validators=[django.contrib.auth.validators.UnicodeUsernameValidator()], verbose_name='Никнейм'),
        ),
    ]
